ui/helpers/utils/conversions.util.js

```diff
@@ -135,6 +135,15 @@ export function decGWEIToHexWEI(decGWEI) {
   });
 }
 
+export function decWEIToHexWEI(decWEI) {
```

Comment:
We should just be able to use the `decimalToHex` function, already in this file, instead of this new one.